crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -1329,7 +1343,7 @@ mod tests {
 
     #[async_test]
     async fn one_time_key_creation() -> Result<()> {
-        let account = ReadOnlyAccount::new(user_id(), device_id());
+        let account = ReadOnlyAccount::with_device_id(user_id(), device_id());
```

Comment:
Would have been nice for easier reviewing if renaming `new` to `with_device_id` was a separate commit.

Reply:
It is https://github.com/matrix-org/matrix-rust-sdk/pull/2349/commits/0107d4710f45b9ee13964029454ddf38e7ace0eb.

Reply:
No, that introduces new constructors too, which makes sense to look at in review as opposed to just the rename, that I would have skimmed over very fast if that was its own commit.

Reply:
Ah indeed, the new constructor is unused in that patch. Sorry.